docs/conf.py

```diff
@@ -152,3 +146,18 @@ def setup(app):
     lines = lines.replace(prefix, newPath)
     with open('index.md', 'wt') as f:
         f.write(lines)
+
+    # placeholder 
+    # actual url: https://raw.githubusercontent.com/The-OpenROAD-Project/OpenROAD/master/docs/contrib/GitGuide.md
+    url = "https://raw.githubusercontent.com/The-OpenROAD-Project/OpenROAD/9409c25c95c5f91a3ad7ba293c4529f39822ea57/docs/contrib/GitGuide.md"
```

Comment:
@vvbandeira Is that ok or point to master will be good?

Reply:
@vijayank88 It will work for now, then when OR docs is merged we can change the link